packages/tools/locales/fr_FR.po

```diff
@@ -4113,7 +4113,7 @@ msgstr "date de modification"
 
 #: packages/lib/Synchronizer.ts:189
 msgid "Updated local items: %d."
-msgstr "Objets mis à jour localement : %d."
+msgstr "Objets local mis-à-jour : %d."
```

Comment:
"m-à-j" to make it shorter

Reply:
if we go for abbreviation then maybe just "màj" ?  It'is quite common and more readable imho.

Reply:
yes màj is fine